projects/angular/src/layout/_card.clarity.scss

```
@@ -110,26 +111,31 @@
     @include mixins.fixForMsEdge() {
       .dropdown > .dropdown-toggle::after {
         display: inline-block;
-        margin-top: calc(-1 * tokens.$cds-global-space-6);
+        margin-top: calc(-1 * density.$clr-base-vertical-offset-l);
       }
     }
   }
 
   .card-block {
-    padding: tokens.$cds-global-space-7;
+    padding: density.$clr-base-vertical-offset-xl density.$clr-base-horizontal-offset-xl;
   }
 
   .card-footer {
     display: flex;
     flex-direction: row;
-    gap: tokens.$cds-global-space-5;
-    padding: tokens.$cds-global-space-5 tokens.$cds-global-space-7;
+    gap: density.$clr-base-gap-s;
+    padding: density.$clr-base-vertical-offset-m density.$clr-base-horizontal-offset-xl;
   }
 
   h3.card-header,
   .card-header {
-    @include mixins.generate-typography-token('SECTION-20-STD');
-    padding: tokens.$cds-global-space-7;
+    @include mixins.generate-typography-token(
+      'SECTION-20-STD',
+      (
+        line-height: density.$clr-base-typography-line-height-20,
+      )
+    );
```

Override of line heigth looks like an error in figma. Confirm it with designers.
```suggestion
    @include mixins.generate-typography-token('SECTION-20-STD');
```